common/src/main/java/no/difi/meldingsutveksling/noarkexchange/receive/PayloadConverterImpl.java

```diff
@@ -54,7 +54,11 @@ public String marshallToString(T message) {
         try {
             Marshaller marshaller = jaxbContext.createMarshaller();
             marshaller.marshal(new JAXBElement<>(new QName(namespaceUri, localPart), clazz, message), sw);
-            return sw.toString();
+            // TODO: Quickfix due to demo 30.05.17 - possible bug in ephorte? See jira issue MOVE-259
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1135)